vignettes/sensors.Rmd

```diff
@@ -158,7 +197,7 @@ For more information on how this data was collected please look into the Parkins
 
 #### Data format and Sample data
 
-`mhealthtools` already comes preloaded with sample tapping data. You can access the data through `data(tap_data)`. The first five rows are shown below, to give you an idea of the structure of the data (this is the input format to the functions that extract gyroscope features). Where `t` is the timestamp; `x` and `y` are the co-ordinates of the tap, and `buttonid` is to indicate whether the user tapped the left or the right button.
+`mhealthtools` already comes preloaded with sample tapping data. You can access the data through `data(tap_data)` or `mhealthtools::tap_data`. The first five rows are shown below, to give you an idea of the structure of the data (this is the input format to the functions that extract gyroscope features). Where `t` is the timestamp; `x` and `y` are the co-ordinates of the tap, and `buttonid` is to indicate whether the user tapped the left or the right button.
```

Comment:
Github isn't allowing me to comment on the relevant section, but it's not clear to me what `depress_threshold` is doing, specifically because I'm not understanding what kind of metric "intertap distance" is supposed to be measuring.

Reply:
I will elaborate on both `depress_threshold` and 'intertap distance'